code/src/components/AlbumArtist.js

```diff
@@ -0,0 +1,34 @@
+
+/* eslint-disable jsx-quotes */
+/* eslint-disable indent */
+/* eslint-disable block-spacing */
+/* eslint-disable quotes */
+import React from 'react';
+
+const AlbumArtist = (props) => {
+  console.log(props.AlbumArtist.artists);
+
+  // let print = props.AlbumArtist.artists.map((singer) => {return (<h3>{singer.name}</h3>)})
+  const print = props.AlbumArtist.artists.map((singer, index) => {return (<a className="AlbumArtistLink" href={singer.external_urls.spotify} target="_blank" rel="noopener noreferrer"><h3 className="AlbumArtist"> {singer.name} {index < props.AlbumArtist.artists.length - 1 ? "& " : ""}</h3></a>)})
+// // console.log(props.AlbumArtist.artists)
```

Comment:
maybe you could remove the comments that don't serve any purpose to the reader, to clean up the code?